rps_game.js

```diff
@@ -64,4 +64,80 @@ document.getElementById('play').addEventListener('click', function () {
   resetComputerDisplay();
   playGame();
 
-});
+}); */
+
+
+(function () {
+  let userInput = {
+    userGesture () {
+      return document.querySelector('input[name=gesture]:checked').value;
+    },
+    clickToPlay () {
+      document.getElementById('play').addEventListener('click', function () {
+        displayReaction.resetComputerDisplay();
+        displayReaction.playGame();
+      });
+    }
+  };
+
+  let computerMove = {
+    _gestureChoice: ['rock', 'paper', 'scissors'],
+
+    get compGesture() {
+      randomIndex = Math.floor(Math.random() * 3);
```

Comment:
you need a `let` here

Reply:
_No need to change it though, unless you want to update the game to Rock, Paper, Scissors, Lizard, Spock! smile_

I was thinking of that!